iphone/Classes/TiUIiOSProxy.m

```diff
@@ -820,5 +820,11 @@ - (id)createFeedbackGenerator:(id)args
 #endif
 #endif
 
+#ifdef __IPHONE_11
```

Comment:
There is no macro __IPHONE_11 . It should be  __IPHONE_11_0.
Better is we use our own defined macro IS_XCODE_9.